Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.fedoraproject.org/fedora:41 AS builder
+FROM registry.fedoraproject.org/fedora:42 AS builder
```

Comment:
@dustymabe do we need to change this to quay by any chance?

Reply:
yeah. let's do that too

Reply:
done, I have changed it to quay